doc/user/repositories.xml

```diff
@@ -472,6 +472,18 @@ int main() { printf("Hello, world!\n"); }
 
     </para>
 
+    <para>
+
+    Note that while other files are searched through the chain of
+    repositories, &SConstruct; is special - it must be found either in
+    the current directory or the first directory
+    specified using the <literal>-Y</literal>
+    (or the <literal>--repository</literal> or
+    <literal>--srcdir</literal> synonyms)
+    command line option, or the build will abort.
+
```

Comment:
I ran into this doing an experiment, and it seemed worth mentioning... we don't have to keep.

Reply:
Interesting.  I think it's quite uncommon to specify more than one repository, but definitely worth noting.
Not sure if I wouldn't consider that a bug.. but it's off on the edges so unless it breaks someones expectations.. I think it's ok for now.